ui/src/index.css

```diff
@@ -11,3 +16,14 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+#main-content {
```

Comment:
I got this main-content div from the inspect-element tool, so wasn't sure I could style it directly using useStyles since the classNames have to be given to a div we directly write ourselves. Let me search if and how we could do it using useStyles.

Reply:
Most of the React-Admin (and Material-UI) components accept a `classes` object, that allows you to override its own `useStyles`. Haven't looked into the specific situation here, but this may be achievable

Reply:
Ahh. Let me try.